src/commands/codepush/lib/react-native-utils.ts

```diff
@@ -571,7 +571,11 @@ function getComposeSourceMapsPath(): string {
 
 function getCliPath(): string {
   if (process.platform === "win32") {
-    return path.join(getReactNativePackagePath(), "local-cli", "cli.js");
+    const reactNativeVersion = coerce(getReactNativeVersion()).version;
```

Comment:
If we are removing ms-rest-js in this PR let's move this change to the separate PR. It seems irrelevant.